test/e2e/tests/address-book.spec.js

```diff
@@ -120,4 +121,96 @@ describe('Address Book', function () {
       },
     );
   });
+  it('Edit entry in address book', async function () {
+    await withFixtures(
+      {
+        fixtures: 'address-entry',
+        ganacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        await driver.clickElement('.identicon__address-wrapper');
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.clickElement({ text: 'Contacts', tag: 'div' });
+        await driver.clickElement('[data-testid="recipient"]');
+
+        await driver.clickElement({ text: 'Edit', tag: 'button' });
+        const inputUsername = await driver.findElement('#nickname');
+        await inputUsername.clear();
```

Comment:
I don't think we need to use the `clear` method as the `fill` method that follows also clears the input under the hood.

Reply:
Done